keyboards/splitography/readme.md

```diff
@@ -0,0 +1,42 @@
+# Splitography
+
+![Splitography](https://cdn.shopify.com/s/files/1/1092/5780/products/IMG_20171117_122316-01_large.jpg)
```

Comment:
Dead link.

Reply:
Thanks. It seems the official website was updated recently.